dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -18,9 +18,17 @@
 
 _tf_plugins = glob.glob(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'libdali_tf*.so'))
 _dali_tf_module = None
-for _libdali_tf in _tf_plugins:
+# move plugin build aganist nightly to the end of list
+_processed_tf_plugins = list(filter(lambda x: 'nightly' not in x, _tf_plugins))
+for elm in _tf_plugins:
+  if 'nightly' in elm:
+    _processed_tf_plugins.append(elm)
+
+for _libdali_tf in _processed_tf_plugins:
   try:
     _dali_tf_module = tf.load_op_library(_libdali_tf)
+    if 'nightly' in _libdali_tf:
+      print ('Using DALI plugin compiled aganist nightly build - no other suitable was found for present TensorFlow version')
```

Comment:
If it works it work, right?  We might be able to have this print be only when debugging?